### PR TITLE
Initial Support page and some wording and style changes for index

### LINKED ISSUE #13: Add/Change wording to emphasis that these are community builds
The quality of the binaries we produce will improve over time but we should be clear to consumers that the OpenJDK binaries produced should not be used in Production and that we do not support them professionally.

### PR DESCRIPTION
**Pull request by karianna**
@Joe-Brady Very happy to have a discussion about the impl and the wording, I tried to follow the existing style as much as possible but have probably deviated.

Fixes #13

### CODE REVIEW

File: support.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  <!-- TODO Style should match installation page, it doesn't quite... -->
+  <main class="grey-bg">
+
+    <h1 class="margin-bottom">Support</h1>
+
+    <div id="support-container">
+      <div class="margin-bottom">
+        <h2 class="bold">WARNING!</h2>
+        <p id="notforproduction">
+          The AdoptOpenJDK OpenJDK binaries are <strong>not</strong> fully tested (i.e. JCK/TCK + further testing) and verified. If you want a foramlly tested
```
Comment (tellison) [inline support.html]: typo on "formally".

File: support.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          and verified Java binary then you can get one from vendors such as Oracle, Azul, IBM, RedHat, SAP and others.
```
Comment (tellison) [inline support.html]: "RedHat" -> "Red Hat"

Like acceptance speeches, once you start listing people you are going to have to try and be fair, and comprehensive, or perhaps consider not mentioning anybody.  While it is reasonable to give caveats about this build, perhaps we don't express an opinion on others.

File: support.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        </p>
+      </div>
+
+      <div class="margin-bottom">
+        <h2 class="bold">Community Support</h2>
+        <p id="communitysupport">
+          The AdoptOpenJDK community does not offer any official support.  If you do have questions, comments, want to report a bug or wish to contribute then
```
Comment (tellison) [inline support.html]: Did you mean AdoptOpenJDK community, or OpenJDK community?

File: support.html (hunk 2 above)
Comment (karianna) [inline support.html]: Agreed, I'll remove specific names

File: support.html (hunk 3 above)
Comment (karianna) [inline support.html]: OpenJDK I guess, although specific to *these* builds as vendors who are OpenJDK members may wish to support downstream binaries based on these ones.  Does that make sense?

**Comment by karianna** [comment]
I still need to fix some SCSS issues to fall inline with the contribution guide, update coming soon!

**Comment by karianna** [comment]
OK, I've pushed the changes from Tim's comments and the contributing guide.  Very happy to continue the review discussion though!

File: index.html [hunk 4]
```diff
@@ -19,12 +19,19 @@
...
 
   <main class="grey-bg home-main">
-    <p id="intro">Java™ is a popular open-source programming language.  It is used extensively, from applications running large corporate systems to small computers running personal games.  The code for Java is open source and available at <a href="http://openjdk.java.net" target="_blank">OpenJDK™</a>.</p>
+    <p class="intro">
+      Java™ is the world's leading programming language and platform. The code for Java is <a href="http://openjdk.java.net/legal/gplv2+ce.html">open source</a> and available at
+      <a href="http://openjdk.java.net" target="_blank">OpenJDK</a>™. AdoptOpenJDK provides prebuilt OpenJDK binaries
+      from a fully open source set of <a href="https://github.com/AdoptOpenJDK/openjdk-build">build scripts</a> and infrastructure.
+    </p>
+    <p class="intro">
+      <strong>NOTE:</strong> AdoptOpenJDK binaries are <strong>not</strong> professionally supported, see <a href="./support.html">Support</a> for details.
```
Comment (Joe-Brady) [inline index.html]: Instead of `<strong>NOTE:</strong>`, we could use:

`<i class="fa fa-info-circle" aria-hidden="true"></i>`

**Comment by Joe-Brady** [comment]
I have reviewed all the changes, all LGTM except for my one comment on index.html.

I may add some minor changes at a later date, but I think after the index.html change it's good for now!

File: index.html (hunk 4 above)
Comment (karianna) [inline index.html]: cool, fixed in latest push

**Comment by karianna** [comment]
@tellison I'll wait for your further thoughts about the wording on that support page.

**Comment by karianna** [comment]
@Joe-Brady - Yeah it was because your earlier PR got merged which created a merge conflict.  We shouldn't actually have the dist files in git at all I don't think

**Comment by Joe-Brady** [comment]
@karianna That would be true if we weren't using GitHub Pages as the hosting solution.

However, we are, so the /dist folder does need to be in this repository - it's where the website looks for the files `styles.min.css` and `app.min.js`!

**Pull request merged**